api/desecapi/tests/test_tokens.py

```diff
@@ -102,6 +102,28 @@ def test_update_my_token(self):
                 for k, v in data.items():
                     self.assertEqual(response.data[k], v)
 
+            datas = [
+                {"last_used": "2018-09-06T09:08:43.762697Z"},
+                {"last_used": None},
+            ]
+            for data in datas:
+                response = method(url, data=data)
+                self.assertStatus(response, status.HTTP_200_OK)
```

Comment:
why do we expect 200? Confirm that writes to non-writeable fields get dropped silently